packages/grid/_modules_/grid/hooks/features/selection/useGridSelection.ts

```diff
@@ -253,35 +255,40 @@ export const useGridSelection = (
     }
   }, [apiRef]);
 
+  const handleSingleRowSelection = React.useCallback(
+    (id: GridRowId, event: React.MouseEvent | React.KeyboardEvent) => {
+      const hasCtrlKey = event.metaKey || event.ctrlKey;
+
+      // Without checkboxSelection, multiple selection is only allowed if CTRL is pressed
```

Comment:
> Without checkboxSelection, multiple selection **via click** is only allowed if CTRL is pressed

There's a regression here. I can't select multiple rows with <kbd>Shift</kbd> + click. Try in https://deploy-preview-3275--material-ui-x.netlify.app/components/data-grid/selection/#multiple-row-selection.



![wxWSSCssDd](https://user-images.githubusercontent.com/42154031/144690255-8a91ded9-095a-4128-92c9-614b9ca43d14.gif)

Reply:
Did a test fail for this bug ? If not could we add one ?

Reply:
I added a unit test to verify that with multiple select (checkboxes activated) the Shift+Space allows to select multiple lines